2015/RDO-Meetups.yml

```diff
@@ -4,176 +4,6 @@ tags: RDO
 
 talks:
 
```

Comment:
As we discussed on the call this morning, this was just due to the huge number of meetups each week, and my confusion as to how this works. I think we're on the same page now. I'll just add from here on out, possibly moving to a per-month or per-quarter file if things get cumbersome.